blocks/story-carousel-block/features/story-carousel/default.jsx

```diff
@@ -0,0 +1,154 @@
+import React from "react";
+import PropTypes from "@arc-fusion/prop-types";
+import {
+	Carousel,
+	Image,
+	Icon,
+	Heading,
+	Paragraph,
+	Stack,
+	HeadingSection,
+	getImageFromANS,
+} from "@wpmedia/arc-themes-components";
+import { useContent } from "fusion:content";
+import { useFusionContext, useComponentContext } from "fusion:context";
+import getProperties from "fusion:properties";
+import getTranslatedPhrases from "fusion:intl";
+
+const BLOCK_CLASS_NAME = "b-story-carousel";
+
+const StoryCarousel = ({
+	customFields: {
+		carouselContentConfig: { contentService = "", contentConfigValues = {} } = {},
+		headerText,
+	},
+}) => {
+	const { id } = useComponentContext();
+	const { arcSite } = useFusionContext();
+	const { locale } = getProperties(arcSite);
+	const phrases = getTranslatedPhrases(locale);
+
+	const content =
+		useContent({
+			source: contentService,
+			query: {
+				...contentConfigValues,
+				feature: "story-carousel",
+			},
+			filter: `content_elements{
+      _id
+      description {
+        basic
+      }
+      headlines {
+        basic
+      }
+      promo_items {
+        type
+        url
+        lead_art {
+          type
+          promo_items {
+            basic {
+              type
+              url
+            }
+          }
+        }
+        basic {
+          type
+          url
+        }
+      }
+      websites {
+        ${arcSite} {
+          website_url
+        }
+      }
+    }`,
+		}) || [];
+
+	const { content_elements: elements = [] } = content;
+	if (!elements.length) return null;
+
+	return (
+		<Stack className={BLOCK_CLASS_NAME}>
+			{headerText && headerText.trim() !== "" ? (
+				<HeadingSection>
```

Comment:
See this PR - https://github.com/WPMedia/arc-themes-blocks/pull/1320 for an example of how the `<HeadingSection>` wrapping of the block should be

Reply:
I added it.